### DISCUSSION TITLE
My phone suddenly flagged Breezy Weather as Malware

### DISCUSSION THREAD

**Discussion by CatBoi3** [breezy-weather/breezy-weather#1267] [Q&A] [answered]
Idk what happened, but I suddenly found my phone telling me to uninstall breezy weather because it contains malware. 

I had breezy weather for a long time before this and didn't even update it. I uninstalled and reinstalled it and it says there's nothing wrong with it now.

Anyone have a clue on what happened?

**Comment by papjul** [comment]
I don't know, please send us the full report and we will tell you what the part of the code they detected as malware does.

Since it's open source, you can also do the analysis by yourself:
1. Download the mapping file from the Releases page: https://github.com/breezy-weather/breezy-weather/releases (make sure to select the correct flavor and the correct version detected as malware)
2. Extract it
3. Find a match of the class reported as malware in the mapping file, and read the source code of that class: https://github.com/breezy-weather/breezy-weather
4. See by yourself if it does anything wrong

If the newer version is not detected as malware, this could also hint to your app detecting old versions of apps as malware to force you to update your apps, so you don't keep a potentially vulnerable version on your device.

**Reply by CatBoi3** [reply]
The report is gone and I forgot to take a picture. As far as I can remember it said it found an apk file with the word " malware" in its name. 

I'm sorry I can't provide more context. If this happens again I'll send a picture.

**Reply by papjul** [reply]
That's a curious finding, but the APKs you download from our GitHub releases don't have "malware" in their filename, so I don't think it's related

**Reply by CatBoi3** [reply]
I downloaded mine from f-droid if that's relevant.

**Reply by papjul** [reply]
Your F-Droid app probably deletes the APK once the app is installed (it would take twice the storage otherwise), so probably something else

**Reply by CatBoi3** [reply]
Alright. Maybe it was an error on the antivirus' part. If it happens again, I'll send a pic of the report here.

**Reply by papjul** [reply]
I opened #1268 which may or may not be related to what you report (it's not about a filename)

**Reply by CatBoi3** [reply]
I think that actually hits the problem on the mark. My phone's antivirus is based on avast mobile and that's probably the same thing that was flagged

**Reply by LeavingBeta** [reply]
Today my F-Secure antivirus marked the Breezy Weather app as infected as well

**Comment by papjul** [comment, marked as answer]
If you still experience issues and found this topic, please follow instructions from #1268
